contrib/hook.sh

```diff
@@ -4,10 +4,5 @@ echo "WASP SWAP hook"
 
 set -x
 
-CG_PATH=$(jq -er '.linux.cgroupsPath' < config.json)
-
-if [[ "$CG_PATH" = *burst* ]];
```

Comment:
Why is this conditional dropped?

Reply:
We don't need to condition since the hook JSON configuration already does the filtering for us, based on the virt-launcher pod annotation.

Reply:
@fabiand From what I remember even the high-performance VMs are created under Burstable pod QOS

Reply:
I do not think so. Guaranteed VMs are using guaranteed class.

Reply:
@fabiand OK so lets restore the condition

Reply:
Thx